### ISSUE TITLE
Issue Reporter: show a good issue title example for placeholder

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#43472]
Testing -210874

Could we do better than `Please enter a title.`? Perhaps an example that feels somewhat realistic to convey what a good issue title looks like? `E.g. missing alt text on extension readme image` for an extension issue. We could use different ones for each source.

![Image](https://github-com/microsoft/vscode/assets/25163139/bc2fc98f-b42d-4be9-8e13-296759d5b610)
